ant-cli/src/main.rs

```diff
@@ -34,7 +35,9 @@ use tracing::Level;
 async fn main() -> Result<()> {
     color_eyre::install().expect("Failed to initialise error handler");
     let opt = Opt::parse();
-    if let Some(network_id) = opt.network_id {
+    if opt.peers.alpha {
+        version::set_network_id(ALPHANET_ID);
+    } else if let Some(network_id) = opt.network_id {
         version::set_network_id(network_id);
     }
```

Comment:
I'm not sure how these lines crept in, as it was removed in this PR and it is now set inside the `autonomi` lib, rather than at the cli. https://github.com/maidsafe/autonomi/pull/2914/files#diff-6601c049ff7babe23590437ea97dc668f98f20b261768b146788a7019663888aR220

This change has to be moved inside `Autonomi::init_with_config()`